contributors/design-proposals/storage/container-storage-interface.md

```diff
@@ -441,37 +441,143 @@ Alternatively, deployment could be simplified by having all components (includin
 
 ### CSI Credentials
 
-This part of proposal is not going to be implemented in alpha release.
+CSI allows specifying credentials in CreateVolume/DeleteVolume, ControllerPublishVolume/ControllerUnpublishVolume, and NodePublishVolume/NodeUnpublishVolume operations.
```

Comment:
add NodeStage (and Unstage?) operation pairs here?

Reply:
Good catch, fixing.